app/assets/javascripts/short-form/components/preferenceComponent.js.coffee

```diff
@@ -52,7 +52,7 @@ angular.module('dahlia.components')
 
       # For 588 Mission preference, we need to override some defaults.
       @isEmploymentDisability = =>
-        @title == 'Employment/Disability Preference'
+        @title == 'Employment or Disability Preference'
 
       @descriptionToTranslate = =>
         if @isEmploymentDisability() then "e7b_custom_preferences.employment_disability.description" else @translatedDescription
```

Comment:
this isn't being recognized by grunt when looking for translations. need to use translate.instant or translate() or something to flag it so it isn't deleted

Reply:
I fixed this in #1456